test/scsi_defs_test.cc

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+#include "lib/scsi_defs.h"
+
+/*
+  Tests the ControlByte class
```

Comment:
I'm not sure if we want all the `scsi_defs` tests to go in this file or if we want separate `.cc` tests for every class. In that case, I'll change the name of this file to `control_byte_test.cc`